src/App.scss

```diff
@@ -28,6 +28,7 @@
   }
 
   .pf-c-card > :last-child.pf-c-card__body {
+    padding-top: 0;
```

Comment:
It looks like this has caused the `Add fact` button on the EditBaseline.js page to touch the top of the card. May just need to add margin-top to the `display-margin` class, but I'm bad at CSS and you may know a better method.

Reply:
Fixed